product_docs/docs/pem/8/considerations/setup_ha_using_efm.mdx

```diff
@@ -6,13 +6,10 @@ redirects:
 - /pem/latest/pem_ha_setup/setup_ha_using_efm/
 ---
 
-!!! Note
-    This document walks you through setting up the Failover manager for PEM Server with new installation and not with existing installation.
-
-Postgres Enterprise Manager (PEM) helps database administrators, system architects, and performance analysts to administer, monitor, and tune Postgres database servers.
-
 Failover Manager is a high-availability tool from EDB that enables a Postgres primary node to failover to a standby node during a software or hardware failure on the primary.
-    
+
+You can set up Failover Manager only for the Postgres Enterprise Manager (PEM) server with a fresh installation. You can't set it for an existing PEM installation.
```

Comment:
Sorry, but this still reads weird to me a bit. How about"

```suggestion
You can set up Failover Manager only with a fresh installation of Postgres Enterprise Manager (PEM). You can't set it for an existing PEM installation.
```
If we need to specify that it is the PEM server:
```suggestion
You can set up Failover Manager only with a fresh installation of a Postgres Enterprise Manager (PEM) server. You can't set it for an existing PEM installation.
```